lib/mysql-writer.js

```diff
@@ -65,6 +65,7 @@ function insertRows(tableName, parsedData, connection) {
 
 function writer(file, parsedData, insertingCase) {
     let tableName = convertFileNameToTableName(file);
+    parsedData = Array.isArray(parsedData) ? parsedData : [parsedData];
```

Comment:
This logic would be better off inside XML parser. The parsers should always return a consistent data format.